### PR TITLE
Fix JakkSergal spawn issues

### PR DESCRIPTION
**Pull request by Arokha**

### CODE REVIEW

**Comment by Jakksergal** [comment]
Finally. I've been complaining about this since BS12 rebased off of TG.

**Comment by Jakksergal** [comment]
Hold on a moment, you forgot to put this into a _vr.dm file type. For shame.

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -518,6 +518,9 @@
 
 	new_character.key = key		//Manually transfer the key to log them in
 
+	if(key == "JakkSergal")
```
Comment (Novacat) [inline code/modules/mob/new_player/new_player.dm]: Needs a //VOREStation Edit

Comment (Arokha) [reply code/modules/mob/new_player/new_player.dm]: Fixed

**Comment by Arokha** [comment]
Will rewrite better at a later date

**Pull request closed without merging**